core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/FPCPoolBalancer.scala

```diff
@@ -46,7 +46,8 @@ class FPCPoolBalancer(config: WhiskConfig,
     extends LoadBalancer {
 
   private implicit val executionContext: ExecutionContext = actorSystem.dispatcher
-  private implicit val requestTimeout: Timeout = Timeout(5.seconds)
+  // This value is given according to the total waiting time at QueueManager for a new queue to be created.
+  private implicit val requestTimeout: Timeout = Timeout(8.seconds)
```

Comment:
I just increased this to align it with the retry timeout at the queue manager.
https://github.com/apache/openwhisk/blob/master/core/scheduler/src/main/scala/org/apache/openwhisk/core/scheduler/queue/QueueManager.scala#L421

In the default configuration, the queue manager waits for a new queue to be created for around 8 seconds.

Regarding this value, it should not be too high because if a target scheduler becomes unresponsive, the controller should try to create a queue against another healthy scheduler.

This change might need to be reverted because controllers would retry to create queues after 8 seconds but the activations would already be dropped by the queue manager.